client/app/components/Tooltip/index.jsx

```diff
@@ -26,8 +26,8 @@ export const Tooltip = (props: Props): Node => {
   const tooltipId = id || `tooltip${Utils.randomString()}`;
   return (
     <div className={css.tooltipWrapper}>
-      <div className={css.tooltipElement} aria-labelledby={tooltipId}>
-        {Utils.renderContent(element)}
+      <div className={css.tooltipElement}>
+        <div aria-labelledby={tooltipId}>{Utils.renderContent(element)}</div>
```

Comment:
This doesn't fix the problem [outlined in the issue](https://github.com/ifmeorg/ifme/issues/2120) since `aria-labelledby` is still being applied to `div` which has no default value for `role`. We'll want to pass in an optional argument into [Utils.renderContent()](https://github.com/ifmeorg/ifme/blob/8201c2f1923a57e303a7eccb66173c73c5f1052c/client/app/utils/index.js#L36) that lets us apply custom props or HTML attributes to the `element`.

For this util function, we use a third-party dependency called [render-react-html](https://github.com/hatashiro/react-render-html). It looks like it's no longer maintained, so we likely want to write something more custom to be able to render HTML as a React element and ALSO apply custom props / HTML attributes.

Ahh, so I realize this may not be a `beginner friendly` issue as it was originally labelled as 😅  Let me know if you still want to work on it, happy to work on it together as well!

Reply:
Looks like the prop type for `element` is `any` (which is not the best decision 😅 ), but that very likely means it could be a React `Node` or a `string`. I think if it's a `Node`, you should be able to do something like `element.aria-labelledby = tooltipId;`. 

If it's a `string`, then the changes I mentioned above is the way to go.

Reply:
Oh haha, yes, I attempted this issue as a beginner in React and took this on because of the beginner friendly label. 

I'm definitely willing to work on it together. I have sent an email to join the slack channel.

Reply:
Let's sync on this on Slack then!